src/edu/csus/ecs/pc2/list/SubmissionSolutionList.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.list;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Hashtable;
+import java.util.List;
+
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+
+/**
+ * List of judge's solutions
+ *
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public class SubmissionSolutionList extends ArrayList<SubmissionSampleLocation> {
+
+	/**
+	 *
+	 */
+	private static final long serialVersionUID = 6231245379952020474L;
+
+	private Hashtable<String, String> submissionDirectoryToAcronym = new Hashtable<String, String>();
+
+	public SubmissionSolutionList() {
+		super();
+        createSubmissionDirectoryMap();
+	}
+
+	public SubmissionSolutionList(IInternalContest mycontest, String cdpPath) {
+		super();
+		createSubmissionDirectoryMap();
+		loadCDPSampleSubmissionList(mycontest, cdpPath);
+	}
+
+	private void createSubmissionDirectoryMap() {
+        // Create quick access to look up acronym for folder
+        for(String [] subFolderRow : SubmissionSampleLocation.CLICS_SUBMISSION_TO_ACRONYM) {
+            submissionDirectoryToAcronym.put(subFolderRow[0], subFolderRow[1]);
+        }
+	}
```

Comment:
this is indented wrong

Reply:
Fixed and pushed. There were residual tabs from the original code.